hphp/runtime/vm/jit/vasm-simplify.cpp

```diff
@@ -196,6 +196,37 @@ bool simplify(Env& env, const setcc& vsetcc, Vlabel b, size_t i) {
   });
 }
 
+template <typename Reg>
+bool operand_one(Env& env, Reg op) {
```

Comment:
@mxw - I'm not sure I understand your comment about csinc{} above.  The csoneb still has
to go through the rest of the passes to get real registers.  I don't think you can make it opaque to the other passes.  I can move the actual peep into the new arm simplify file.